packages/common/src/tokens/utils.ts

```diff
@@ -24,3 +56,86 @@ export function sortByAddress<T extends Token | TokenObject | TokenAmount | Toke
     return address.toLowerCase();
   });
 }
+
+const basicTokenMap: Record<number, Record<string, Token>> = {
+  [ChainId.mainnet]: mainnetTokens,
+  [ChainId.avalanche]: avalancheTokens,
+  [ChainId.polygon]: polygonTokens,
+  [ChainId.optimism]: optimismTokens,
+  [ChainId.arbitrum]: arbitrumTokens,
+  [ChainId.metis]: metisTokens,
+  [ChainId.gnosis]: gnosisTokens,
+  [ChainId.zksync]: zksyncTokens,
+  [ChainId.base]: baseTokens,
```

Comment:
Please add BNB to this list and sort the entries by chain ID to facilitate a clearer code review.

Reply:
added

Reply:
It's still not be sorted yet.

Reply:
sorted